src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -95,18 +92,18 @@ const Button = styled.button.attrs<ButtonProps>((props: ButtonProps) => ({
   flex-direction: row;
   justify-content: center;
   align-items: center;
-  border: none;
   background: ${({ $active, theme }: ButtonProps) =>
     $active
       ? theme.click.button.group.color.background.active
       : theme.click.button.group.color.background.default};
   color: ${({ theme }) => theme.click.button.group.color.text.default};
   font: ${({ theme }) => theme.click.button.group.typography.label.default};
-  padding: ${({ theme }) => theme.click.button.basic.space.y}
-    ${({ theme }) => theme.click.button.basic.space.x};
-  gap: ${({ theme }) => theme.click.button.basic.space.group};
+  padding: ${({ theme, $type = "default" }) =>
+    `${theme.click.button.group.space[$type].y} ${theme.click.button.group.space[$type].x}`};
   ${({ $fillWidth = false }) => ($fillWidth ? "flex: 1;" : "")};
+  border-radius: ${({ theme }) => theme.click.button.group.radii.all};
```

Comment:
we're not using `theme.click.button.group.radii.end` anymore, should we remove it?
Also, do css variables sync with the tokens in some way, or we need to update them manually?